frontend/src/views/Register.vue

```diff
@@ -57,17 +68,30 @@
   </v-container>
 </template>
 
+
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator'
 import config from '@/config'
 @Component
 export default class Register extends Vue {
   private passwordShow: boolean = false
+  private cPasswordShow: boolean = false
   private valid: boolean = false
   private username: string = ''
   private password: string = ''
+  private secretKey: string = ''
+  private accessKey: string = ''
   private register() {
-    const payload = { email: this.username, password: this.password }
+    const payload = { email: this.username, password: this.password,
+    secret_key: this.secretKey, access_key: this.accessKey}
```

Comment:
Please indent this properly, and add a space after the last bracket.
So something like
```js
const payload = {
  email: this.username,
  password: this.password,
  secret_key: this.secretKey,
  access_key: this.accessKey,
}
```